releasenotes/notes/implement-django-postgres-dbm-trace-propagation-6ed725516d8bc5c1.yaml

```diff
@@ -0,0 +1,5 @@
+---
+features:
+  - |
+    For new features such as a new integration or component. Use present tense with the following format:
```

Comment:
did you mean to write a release note?

Reply:
yes, forgot to push the commit, pushing it now